### PR TITLE
Update app id name to customImportMapDashboards

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
Update app id name to customImportMapDashboards from customImportMap, which is required from infra team.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: opensearch_dashboards.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "id": "customImportMap",
+    "id": "customImportMapDashboards",
```
Comment (VijayanB) [inline opensearch_dashboards.json:2]: Is it safe to rename plugin id? cc: @ashwin-pc

**Review by heemin32 — APPROVED** [review]

File: opensearch_dashboards.json (hunk 1 above)
Comment (junqiu-lei) [reply opensearch_dashboards.json:2]: Actually, we are using customImportMapDashboards at released branches and 2.x, but just not in main branch. So I think we need keep consistent for released usage. cc @peterzhuamazon 

https://github.com/opensearch-project/dashboards-maps/blob/2.4/src/plugins/custom_import_map/opensearch_dashboards.json#L2

https://github.com/opensearch-project/dashboards-maps/blob/2.4/src/plugins/custom_import_map/opensearch_dashboards.json#L2

**Review by peterzhuamazon — APPROVED** [review]

**Comment by navneet1v** [comment]
@junqiu-lei with this id change and name change I hope plugin name will be consistent. So, it won't break any BWC.

**Comment by navneet1v** [comment]
> @junqiu-lei with this id change and name change I hope plugin name will be consistent. So, it won't break any BWC.

Please ignore found the response here: https://github.com/opensearch-project/dashboards-maps/pull/179#discussion_r1065182723

**Review by VijayanB — APPROVED** [review]

**Comment by junqiu-lei** [comment]
> @junqiu-lei with this id change and name change I hope plugin name will be consistent. So, it won't break any BWC.

In all released branches, it uses the "customImportMapDashboards", not sure why it used "customImportMap" only in main branch. I think the name here is mainly used for infra build work.

**Pull request merged**